app/controllers/restaurants_controller.rb

```diff
@@ -8,6 +8,11 @@ def create
     @restaurant = Restaurant.new(restaurant_params)
     @cuisine = Cuisine.find(params[:cuisine_id])
     @restaurant.cuisines << @cuisine
+    @restaurant_photo = RestaurantPhoto.new
+    @restaurant_photo.user = current_user
+    @restaurant_photo.restaurant_photo = Rails.root.join("spec/support/images/stock-restaurant.jpg").open
+    @restaurant_photo.title = "Some Restaurant"
+    @restaurant.restaurant_photos << @restaurant_photo
```

Comment:
Dude.....this is hard coding....dude
Also, why do you have two create methods on the API restaurants controller and the regular one. Only one of them needs to exist.

Reply:
Yes, you are right, we moved functionality for this form to react, this is no longer needed, will fix

Reply:
removed everything except a show method in the RestaurantsController that just renders a <div id="app"/> that react can latch on to